.github/workflows/http.yml

```diff
@@ -31,17 +31,17 @@ jobs:
             {
               arch: x86_64,
               alby_arch: x86_64-unknown-linux-gnu,
-              os: ubuntu-20.04,
+              os: ubuntu-24.04,
```

Comment:
if we do this, will the build still work on 22.04?

Reply:
Whoops, missed that 22.04 is still supported